test/fixtures_identity_store.js

```diff
@@ -0,0 +1,181 @@
+// Identity Personas
+
+// NURSECONNECT
+    // (+27820001001) - new identity; no subscriptions
+    // (+27820001002) - existing identity with an active MomConnect subscription (no active NurseConnect subscription)
+    // (+27820001003) - existing identity with an active NurseConnect subscription
+
+
+module.exports = function() {
+    return [
+
+        // 0: get identity by msisdn +27820001001
+        {
+            "request": {
+                "url": 'http://is.localhost:8001/api/v1/identities/search/',
+                "method": 'GET',
+                "params": {
+                    "details__addresses__msisdn": '+27820001001'
+                },
+                "headers": {
+                    "key": "get.identity.new.27820001001"
```

Comment:
@imsickofmaps Where would you put this `key` in the fixture? I would have put it on the same level as, and just before `"request"`